app/src/androidTest/java/ch/epfl/qedit/util/LocationHelper.java

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.qedit.util;
+
+import android.content.Context;
+import android.location.Criteria;
+import android.location.LocationManager;
+import ch.epfl.qedit.backend.location.MockLocService;
+
+public class LocationHelper {
+    public static void initLocationProvider(Context context) {
+        LocationManager manager =
+                (LocationManager) context.getSystemService(Context.LOCATION_SERVICE);
+
+        if (!manager.isProviderEnabled(MockLocService.LOCATION_PROVIDER)) {
+            manager.addTestProvider(
+                    MockLocService.LOCATION_PROVIDER,
+                    false,
```

Comment:
I think it would be great to explain what this army of `falses` is.

Reply:
Yeah, you're right. In android studio, you can see the name of the arguments, which makes it clearer, but not on GitHub, so I will add it.